code/datums/uplink_item.dm

```diff
@@ -576,8 +576,8 @@ var/list/uplink_items = list()
 	cost = 1
 
 /datum/uplink_item/device_tools/bugdetector
-	name = "Bug Detector"
-	desc = "A functional multitool that can detect certain surveillance devices. Its screen changes color if the AI or a pAI can see you, or if a tape recorder or voice analyzer is nearby. Conspicuous if currently detecting something. Examine it to see everything it detects. "
+	name = "Bug Detector w/Camera Disabler built-in"
```

Comment:
How about just "Bug Detector & Camera Disabler"?

Reply:
Doesn't that sound like it's a bundle/it's more than just 1 item? It crossed my mind and that's the reason I didn't choose that name. I wouldn't mind changing it though.

Reply:
"Combination Bug Detector and Camera Disabler"